next/schema-generator/definitions/registracia-predzahradky.ts

```diff
@@ -0,0 +1,247 @@
+import {
+  conditionalFields,
+  conditionalStep,
+  fileUpload,
+  input,
+  markdownText,
+  object,
+  radioGroup,
+  schema,
+  select,
+  step,
+  textArea,
+} from '../generator/functions'
+import { createCondition, createStringOptions } from '../generator/helpers'
+
+enum TypRegistracie {
+  NOVA = 'nova',
+  EXISTUJUCA = 'existujuca',
+}
+
+const prilohy = (typRegistracie: TypRegistracie) => [
```

Comment:
I pushed for product to make the same step for both of the cases, as this is hacky.